recipes/mysql-connector-cpp/all/conanfile.py

```diff
@@ -0,0 +1,49 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake
+
+
+class MysqlConnectorCppConan(ConanFile):
+    name = "mysql-connector-cpp"
+    version = "8.0.27"
```

Comment:
the `version` attribute is not needed as the CI service will build the versions listed in `config.yml`